CONTRIBUTING.md

```diff
@@ -38,6 +38,8 @@ Follow these steps and note these guidelines to begin contributing:
 
 ## Working on an issue
 
+- As a first-time contributor, please do not request assignment to more than one issue at a time, except in rare scenarios where the issues are closely related.
+- As a returning contributor, ensure that you do not have an open pull request before requesting assignment to a new issue. Your pull request from a previous issue must be reviewed and merged first.
```

Comment:
I don't think that it's a problem that you work on multiple issues. I think the problem arises when second PR references changes in the first PR...

For instance: 

- A USER edits a file
- The same USER wants to submit another PR but is working on the same branch as the file with changes. 
- The MAINTAINER requests changes on the first PR which now need to be made on the second PR.

Reply:
gotcha!